boot/zephyr/main.c

```diff
@@ -69,7 +70,8 @@ void main(void)
 	irq_lock();
 	_MspSet(vt->msp);
```

Comment:
We probably should have these log and clock calls before the _MspSet.

Did you find specific problems with the clock enabled, since the interrupts should be masked?